topics/introduction/tutorials/galaxy-intro-jupyter/tutorial.md

```diff
@@ -0,0 +1,124 @@
+> 3. Click *Launch*
+>
+{: .hands_on}
+
+# Install Libraries in Jupyter
+
+You can install tools and libraries in Jupyter through conda and pip. In this tutorial we are going to use two libraries, [pandas]() and [seaborn]() respectively allowing to manipulate data as Dataframe and to create graphs.
+
+> ### {% icon hands_on %} Hands-on: Install from a Conda recipe
+>
+> 1. Click on a cell of your notebook to edit it (verify that it is defined as a `Code` cell)
+> 2. Enter the following lines : `!conda install -y pandas` and `!conda install -y seaborn`
+>   - The `!` indicate you are typing a bash command line (alternatively you can use `%%bash` at the beginning of your cell )
+>   - The `-y` option allows the installation without asking for confirmation  (The confirmation is not managed well by notebooks)
+> 3. `shift`+`return` to run the cell or click on the run cell button.
+>
+{: .hands_on}
+
+
+> ### {% icon hands_on %} Hands-on: Import Python libraries
+>
+> 1. Click on a cell of your notebook to edit it (verify that it is defined as a `Code` cell)
+> 2. Enter the following lines : `import pandas as pd`, `import seaborn as sns`, `from IPython.display import display`, and `import matplotlib.pyplot as plt`.
+> 3. `shift`+`return` to run the cell or click on the run cell button.
+>
+{: .hands_on}
+
+
+# Graph Display in Jupyter
+
+In this tutorial we are going to simply plot a distribution graph of our data.
+
+> ### {% icon hands_on %} Hands-on: Draw a distribution plot
+>
+> 1. Open the dataset as a pandas Dataframe with the function `dataframe = pd.read_table("[file_number]", header=None)`
+>    - The files are referenced in Jupyter by their number in the history.
+> 2. Create your figure with the command `fig, ax = plt.subplots( nrows=1, ncols=1 ,figsize=(15, 10) )`
+>   -  `nrows=1, ncols=1` means you will have one plot in your figure (one row and one column)
+>   -  `figsize` parameter determine the size of the figure
+> 3. Draw the distribution plot of the second column of our dataset with the command `sns.distplot(dataframe[1]);`
+> 4. Show the figure in the Jupyter notebook with `display(fig)` ![](../../images/jupyter_plot.png)
+>
+{: .hands_on}
+
+
+# Import / export Data
+
+In addition of starting a Jupyter notebook with datasets included at the beginning , you can import them later using the `get(12)` command, with the number of your dataset in the history (If you are working on a collection, unhide datasets to see their numbers).
+If you want to save a file you generated in your notebook, use the `put("file_name")` command. That is what we are going to do with our distribution plot.
+
+> ### {% icon hands_on %} Hands-on: Save an Jupyter generated image into a Galaxy History
+>
+> 1. Create an image file with the figure you just draw with the command `fig.savefig('distplot.png')`
+> 2. Export your image into your history with the command  `put('distplot.png')`
+>
+{: .hands_on}
+
+
+# Save the Notebook in your history
```

Comment:
maybe it would also be nice to show how to start jupyter with a previously saved notebook loaded?

Reply:
I talked about it in the first part with how to start a notebook, but I could move it down to another part if you think that makes more sense.

Reply:
oops, missed that, so sorry, it is fine this way, thanks :)